cypress/e2e/OrangeHRM/TC-06_VerifySearchOnEmployeeOnOrangeHRM/TC-06_VerifySearchOnEmployeeOnOrangeHRM.feature

```diff
@@ -0,0 +1,25 @@
+Feature: TC-06 Verify Update Employee On OrangeHRM
+
+   Scenario: 1#: Verify valid search on employee by name
+      Given Common Step: The user login to OrangeHRM
+      And the user navigate to Add Employee page
+      And the user add a new employee
```

Comment:
if you add the employee by API you not need to navigate to the page in the previous step